packages/ignition/src/app/structured-data/action.ts

```diff
@@ -0,0 +1,16 @@
+import { Thing } from ".";
```

Comment:
Do not use import like that, it creates redundant references. Next can handle that, so, it works, but it's not ok.

Using `index.ts` is not an option as well, same issue. Should be `import Thing from './thing'`